test/entrypoint.test.ts

```diff
@@ -265,12 +277,22 @@ describe('EntryPoint', function () {
       const op1 = await fillAndSign({
         initCode: getAccountInitCode(accountOwner1.address, simpleAccountFactory),
         sender: '0x'.padEnd(42, '1'),
-        verificationGasLimit: 1e6
+        verificationGasLimit: 3e6
       }, accountOwner1, entryPoint)
       await expect(entryPoint.callStatic.simulateValidation(op1))
         .to.revertedWith('AA12 initCode must return sender')
     })
 
+    it('should report failure on insufficient verificationGas (OOG) for creation', async () => {
+      const op1 = await fillAndSign({
+        initCode: getAccountInitCode(accountOwner1.address, simpleAccountFactory),
+        sender: '0x'.padEnd(42, '1'),
+        verificationGasLimit: 1e5
+      }, accountOwner1, entryPoint)
+      await expect(entryPoint.callStatic.simulateValidation(op1, { gasLimit: 1e6 }))
+        .to.revertedWith('AA11 initCode failed')
```

Comment:
How do we actually assert that this reverts on OOG? this is a catch-all kind of message that doesn't really check insufficient verificationGasLimit, i.e. the test will falsely pass when failing on any other reason it seems.

Reply:
right. but the same initCode succeeds if gas is enough. EVM code doesn't give you a good way to detect OOG compared to other revert reasons, and I don't want to calculate (fixing revert string to "say failed or OOG", like other items.